modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -863,32 +918,147 @@ If you installed Loki using the Loki Operator, it is advised not to use `querier
 | `object (string)`
 | staticLabels is a map of common labels to set on each flow
 
+`statusTls`
+| `object`
+| TLS client configuration for loki status URL.
+
 | `statusUrl`
 | `string`
-| statusURL specifies the address of the Loki /ready /metrics /config endpoints, in case it is different from the Loki querier URL. If empty, the QuerierURL value will be used. This is useful to show error messages and some context in the frontend
+| statusURL specifies the address of the Loki `/ready`, `/metrics`, and `/config` endpoints, in case it is different from the Loki querier URL. If empty, the `querierURL` value will be used. This is useful to show error messages and some context in the frontend. statusTLS configuration will be used when statusUrl is set.
+
 
 | `tenantID`
 | `string`
-| tenantID is the Loki X-Scope-OrgID that identifies the tenant for each request. it will be ignored if instanceSpec is specified
+| tenantID is the `Loki X-Scope-OrgID` that identifies the tenant for each request. it will be ignored if instanceSpec is specified
```

Comment:
```suggestion
| tenantID is the `Loki X-Scope-OrgID` that identifies the tenant for each request. It will be ignored if instanceSpec is specified.
```

Reply:
hmm where does that come from? I don't see that in the upstream doc : https://github.com/netobserv/network-observability-operator/pull/370/files#diff-6e07f26394c17bac752bf84140a6381b6910ed8900fc739e23965f5d986be7a3R943-R946
🤔
And the part about "instanceSpec" seems wrong, we don't have something called `instanceSpec`

Reply:
Interesting...I wonder if maybe the wording changed here and it just wasn't copied over? 😬 I think the last time we updated the CRD documentation for 1.2, I only did a copy/paste of the parts that changed. I didn't copy the entire file over. SO I wonder if maybe something was missed. I just re-copied the entire file so hopefully everything is there now.